src/lib/rsa/sandboxed/RSAKeysIframe.js

```diff
@@ -0,0 +1,85 @@
+/* global forge */
+
+/**
+ * @typedef {{command: 'generateKey', seed: string, keySize: number}} GenerateKeyCommand
```

Comment:
The `seed` should be better a `Uint8Array` instead of a binary string because it's not very clear that a binary string is expected, and not for example hex encoded data, and binary strings just feel very outdated.
```suggestion
 * @typedef {{command: 'generateKey', seed: Uint8Array, keySize: number}} GenerateKeyCommand
```

Reply:
Resolved in #524